python/practices/practice-3/src/path2/markov_analysis.py

```diff
@@ -69,23 +69,38 @@ def generate_random_text(prefixes, prefix_length, num_words):
     return random_text.strip()
 
 
-def main():
+def perform_markov_analysis_and_generate_text():
```

Comment:
I think `generate_text` is a better name.

Reply:
Done